### DISCUSSION TITLE
Why is Cumulative User Memory so large on Resource Utilization Summary Web

### DISCUSSION THREAD

**Discussion by hanson2021** [trinodb/trino#18637] [Q&A]
(1)Trino Version:403
(2) The Resource Utilization Summary of  trino task from web ui are as following. I found  the value of Cumulative User Memory is very large . How did it statistic?
 
<img width="665" alt="企业微信截图_16917174161894" src="https://github.com/trinodb/trino/assets/80302963/cbcb95fa-9f00-4260-ba92-a3ceb0be67d0">

![图片](https://github.com/trinodb/trino/assets/80302963/f57001b4-4300-429f-b99a-d7a7d1d1bd8e)

(3)My QuerySql:
select 
          t1.query_id,t1.started,t2.physical_input_size,t1.user,t1.query 
        from 
          (select 
            query_id,at_timezone(started,current_timezone()) as started,user,query,
            to_milliseconds(now()-at_timezone(started,current_timezone())) as exectime
          from system.runtime.queries 
          where at_timezone(started,current_timezone())<=date_add('minute',-5,now())
                and state='RUNNING' 
                and query<>'select 1'
          ) t1
        left join
        (
          select query_id,
            sum(if(physical_input_bytes is not null,physical_input_bytes,0.00))/(1024*1024*1024.0) as physical_input_size
            from system.runtime.tasks 
          group by query_id 
         )t2 
       on t1.query_id=t2.query_id
       where t1.exectime/(1000*60)>=10 or t2.physical_input_size>=1024

**Comment by hashhar** [comment]
cc: @raunaqmorarka @sopel39 Seems wrong on a quick look.

@hanson2021 Can you grab the query info JSON for this query?

**Comment by hanson2021** [comment]
> cc: @raunaqmorarka @sopel39 Seems wrong on a quick look.
> 
> @hanson2021 Can you grab the query info JSON for this query?

@hashhar  grab  the query info JSON from Web UI by creating new query
[20230814_015213_07305_2957k.json.txt](https://github.com/trinodb/trino/files/12330969/20230814_015213_07305_2957k.json.txt)

**Comment by sopel39** [comment]
Yeah, that seems wrong.

**Comment by sopel39** [comment]
https://github.com/trinodb/trino/pull/18714 is a fix
